web/config/application.rb

```diff
@@ -15,6 +15,14 @@ class Application < Rails::Application
 
     config.assets.prefix = "/api/assets"
 
+    if ShopifyAPI::Context.embedded?
+      config.action_dispatch.default_headers = {
```

Comment:
Will this override any other defaults that the app might be setting? Can / should we extend whatever value was previously set here?